sigmadsp/application/frontend.py

```diff
@@ -87,7 +87,7 @@ def main():
     if arguments.address is not None:
         backend_address = arguments.address
 
-    response: ControlResponse
+    response: ControlResponse = None
```

Comment:
I wonder why mypy doesn't complain, I would have expected this to be required:
```suggestion
    response: Union[ControlResponse, None] = None
```